apps/project/tasks.py

```diff
@@ -38,3 +39,136 @@ def push_project_to_firebase(project_id: int):
     project_type_handler = get_project_type_handler(project.project_type_enum)(project)
     project_type_handler.push_project_on_firebase()
     return True
+
+
+class SlackMessage:
+    @classmethod
+    async def get_message(
+        cls,
+        input_data: Project,
```

Comment:
Let's break down this input.
Also, we need to rename the get_message method.
Something like get_message_for_project_progress